web/themes/joinup/joinup_theme.theme

```diff
@@ -8,26 +8,37 @@
 use Drupal\Core\Template\Attribute;
 use Drupal\file\Entity\File;
 use Drupal\image\Entity\ImageStyle;
+use Drupal\Component\Utility\Html;
+use Drupal\Component\Utility\Unicode;
 
 /**
  * Implements hook_preprocess_page().
  */
 function joinup_theme_preprocess_page(&$variables) {
-  // Set page title.
   $request = \Drupal::request();
   $route_match = \Drupal::routeMatch();
+
+  // Set page title.
```

Comment:
Documentation here could be improved, saying what we are trying to do with the code here. For example
`// We need to show the header in a blue variant on certain routes`